common/src/main/java/io/homeassistant/companion/android/common/data/keychain/KeyChainRepositoryImpl.kt

```diff
@@ -0,0 +1,58 @@
+package io.homeassistant.companion.android.common.data.keychain
+
+import android.content.Context
+import android.security.KeyChain
+import io.homeassistant.companion.android.common.data.prefs.PrefsRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.security.PrivateKey
+import java.security.cert.X509Certificate
+import javax.inject.Inject
+
+class KeyChainRepositoryImpl @Inject constructor(
+    private val prefsRepository: PrefsRepository
+) : KeyChainRepository {
+
+    private var alias: String? = null
+    private var key: PrivateKey? = null
+    private var chain: Array<X509Certificate>? = null
+    private var isLoading: Boolean = false
+
+    override suspend fun load(context: Context, alias: String) {
+        if (alias == null) return
+
```

Comment:
`alias` is not nullable, so this can be removed

Reply:
Done